conans/test/unittests/tools/scm/test_git_get_commit.py

```diff
@@ -97,3 +97,121 @@ def test_multi_folder_repo():
     c.run_command("git rev-parse HEAD")
     commit_real = str(c.out).splitlines()[0]
     assert new_commit_root == commit_real
+
+def test_relative_folder_repo():
+    def _chdir(new_path):
```

Comment:
I couldn't find a helper on TestClient to do this, and I wanted to minimize use of conan source in unit tests.
Otherwise, I'd just import `chdir` from `conans.tools.files`

Reply:
Yes, ``TestClient`` has a ``chdir()`` ``@contextmanager`` that you can use as ``with myclient.chdir("myfolder"):``

Reply:
nevermind, I see the tools being used in other tests.